File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.py
class Solution:
    def getRow(self, rowIndex: int) -> List[int]:
        l = [1]
        for i in range(rowIndex):
            for j in range(i,0,-1):
                l[j] = l[j] + l[j-1]
            l.append(1)
        return l

File: 0119-pascals-triangle-ii/0119-pascals-triangle-ii.cpp
class Solution {
public:
    double factorial(double n){
        double fact = 1;
        for (int i = 1; i<=n; i++){
            fact*=i;
        }
        return fact;
    }
    double combination(double i, double j){
        return round(factorial(i)/(factorial(j)*factorial(i-j)));
    }
    vector<int> getRow(int rowIndex) {
        vector<int> v;
        for (int i = 0; i<=rowIndex; i++){
            v.push_back(combination(rowIndex,i));
        }
        return v;
    }
};